torax/_src/geometry/pydantic_model.py

```diff
@@ -248,10 +248,53 @@ def build_geometry(self) -> standard_geometry.StandardGeometry:
     )
 
 
+class IMASConfig(torax_pydantic.BaseModelFrozen):
+  """Pydantic model for the IMAS geometry.
+
+  Attributes:
+    geometry_type: Always set to 'imas'.
+    n_rho: Number of radial grid points.
+    hires_factor: Only used when the initial condition ``psi`` is from plasma
+      current. Sets up a higher resolution mesh with ``nrho_hires = nrho *
+      hi_res_fac``, used for ``j`` to ``psi`` conversions.
+    geometry_dir: Optionally overrides the `TORAX_GEOMETRY_DIR` environment
+      variable.
+    equilibrium_object: Either directly the equilbrium IDS containing the relevant data, or the name of the IMAS netCDF file containing the equilibrium.
```

Comment:
can we restrict here to only allow the netCDF file name? Two reasons for this suggestion:

1. simplicity of the interface by having only one option and consistency with the other geometry configs
2. it's not clear to me how the `equilibrium IDS` would be serialised/deserialised

Reply:
I am currently using the direct equilibrium IDS input for the MUSCLE3 actor for Torax. I could change it but would much prefer having this option.